subt/artifacts.py

```diff
@@ -294,7 +294,7 @@ def detect(self, image):
                     f.write(image)
         rcount, w, h, x_min, x_max = count_red(img)
         yellow_used = False
-        if self.is_virtual and rcount < 20:
```

Comment:
Well, this explains why we did not have even single false report (I mean even a single report) of survivor for the "System". Thanks - I would accept it as the minimal viable change